templates/flashes.html

```diff
@@ -0,0 +1,3 @@
+{% for category, message in get_flashed_messages(with_categories=true) %}
+<div class="alert alert-{{ category }}" role="alert">{{ message }}</div>
```

Comment:
Small formatting notes: It would be better if you move second line to the right with few spaces.